playground/lib/home/custom_sheets/attached_floating_bottom_sheet_type.dart

```diff
@@ -0,0 +1,169 @@
+import 'dart:math';
+
+import 'package:flutter/material.dart';
+import 'package:wolt_modal_sheet/wolt_modal_sheet.dart';
+
+class AttachedFloatingBottomSheetType extends WoltModalType {
```

Comment:
Can you ask ChatGpt to add extensive code comment about what this modal type does? What does attaching mean, and what the properties do? You can also explain your own logic.

Reply:
Added some hybrid doc comments. Some custom written some by Copilot

Reply:
Simply amazing! Thanks